lib/k8s/job.js

```diff
@@ -98,6 +99,22 @@ class Job extends EventEmitter {
         const self = this;
 
         debug(`Starting k8Job jobId: ${self.jobId}, runId: ${self.runId}`);
+        const scriptPath = self.command[0];
+        const args = self.command.slice(1);
+        let output;
+
+        if (args.length > 0) {
+            const argumentList = args.map((arg) => `--${arg}`).join(' ');
+            output = `Script used: ${scriptPath} ${argumentList}`;
+        }
+        else {
+            output = `Script used: ${scriptPath}`;
+        }
+
+        if (output && output.length > 0) {
+            iostatus.sendLogOutput(self.runId, output);
+        }
+
```

Comment:
LGTM. 
nit: can we maybe simplify the multiple if-else and if statements? maybe something like this(haven't tested):

```
const scriptPath = self.command[0];
const args = self.command.slice(1);
const argumentList = args ? args.map(arg => `--${arg}`).join(' ') : '';
const output = `Script used: ${scriptPath} ${argumentList}`.trim();

iostatus.sendLogOutput(self.runId, output);

```